tooling-docker/src/main/java/org/jboss/eap/qe/ts/common/docker/Docker.java

```diff
@@ -77,7 +79,31 @@ protected void start() throws Exception {
             }
         });
 
-        containerReadyCondition.waitUntilReady(containerReadyTimeout);
+        long startTime = System.currentTimeMillis();
+        while (!isContainerReady()) {
+            if (System.currentTimeMillis() - startTime > containerReadyTimeout) {
+                throw new Exception("Container was not ready in timeout : " + containerReadyTimeout + " ms");
+            }
+            // fail fast mechanism in case of malformed docker command, for example bad arguments, invalid format of port mapping, image version,...
+            if (!dockerRunProcess.isAlive() && dockerRunProcess.exitValue() != 0) {
```

Comment:
Either one earlier check is to be done on docker process alone or I think I am getting a couple of green tests because of this - in conjunction with expected exception prefix [and suffix] as defined in test classes - even if I have docker daemon not running. 
WDYT?

Reply:
I'm not sure I understand. Do you want to update exception message that docker daemon might not be running?

Reply:
It is not just about the exception message.
The message is the same - i.e. the expected one - that the `testFailFastWithMalformedDockerCommand` method is supposing to catch. 
So my concern is mainly about not getting a green bar from a fail-safe case method related to an unavailable image when instead tests have not been executed, actually - which instead I happen to be able to do.
That said I thought about a pre execution check for docker process to be running. WDYT?

Reply:
Thanks, now i get your point. I'll just rephrase it, we'll add check that docker command is present. This is because there is podman on rhel8 which does not have any docker daemon. 

I've added the check and re-pushed.